shpe-app-web/app/components/Navbar.tsx

```diff
@@ -1,134 +1,79 @@
-'use client';
+'use client'
 
-import Image from "next/image";
-import Link from "next/link";
+import Image from 'next/image';
+import Link from 'next/link';
 import { usePathname, useRouter } from 'next/navigation';
 import { handleLogout } from "@/helpers/auth";
 
 const Navbar = () => {
+
     const router = useRouter();
     const path = usePathname();
 
 
     return (
+        <nav className='bg-[#500000] text-[18px] flex justify-between items-center font-bold h-20 shrink-0'>
 
-        <div className="flex flex-col bg-[#500000] h-screen w-52 justify-between shrink-0 overflow-x-hidden overflow-y-auto">
-            <nav className='flex flex-col w-full text-[18px] font-bold'>
-                <Image src='logo.svg' alt="SHPE Logo" width={200} height={200} className="mb-10" />
-
-                <Link href={'/dashboard'} className={`pr-5 pl-5 hover:shadow-inner-strong relative w-full  h-[45px] pt-2 ${path == '/dashboard' ? 'bg-[#794141]' : 'bg-[#500000]'}`} >
-                    <div className="flex flex-row">
+            <div className='flex justify-evenly w-full '>
+                <Link href={"/dashboard"}>
+                    <div className={`flex  items-center h-full rounded-full pr-5 pl-5 transition duration-300
+                    ${ path =="/dashboard" ?"bg-gray-400" :'hover:bg-gray-400'} `}>
                         <Image src="house-solid.svg" alt="Dashboard" width={20} height={20} />
-                        <div className=" pl-3 "> Dashboard </div>
-                    </div>
+                        <div className='pl-3'> Dashboard </div>
 
-                    {path == '/dashboard' ?
-                        <Image src="Polygon1.svg" alt="Dashboard" width={25} height={20}
-                            style={{ position: 'absolute', top: 9, right: -2, zIndex: 0 }} />
-                        : ''
-                    }
+                    </div>
                 </Link>
 
-                <Link href={'/events'} className={`pr-5 pl-5 hover:shadow-inner-strong relative w-full  h-[45px] pt-2 ${path == '/events' ? 'bg-[#794141]' : 'bg-[#500000]'}`} >
-
-                    <div className="flex flex-row ">
-                        <Image src="calendar-solid.svg" alt="Dashboard" width={20} height={20} />
-                        <div className=" pl-3 "> Events </div>
+                <Link href={"/events"}>
+                    <div className={`flex  items-center h-full rounded-full pr-5 pl-5 transition duration-300
+                    ${ path =="/events" ?"bg-gray-400" :'hover:bg-gray-400'} `}>
+                        <Image src="calendar-solid.svg" alt="Dashboard" width={15} height={20} />
+                        <div className='pl-3 h-fit'> Events </div>
```

Comment:
For each of these title be careful with spacing. For example " Events " is not the same as " Events".  This is not a big deal but will causes weird spacing issue. So ideally truncate the space and make it "Events". This should be done to the other title as well.